omod/src/main/webapp/resources/jsonforms/HMIS-TB-003-TBClientCard-FollowupSection.json

```diff
@@ -0,0 +1,895 @@
+{
+  "name": "DS TB Followup Form",
+  "version": "4.0.0",
+  "description": "This is a followup form for a DS-TB patient",
+  "encounterType": "455bad1f-5e97-4ee9-9558-ff1df8808732",
+  "encounter": "DS TB Encounter (Followup)",
+  "uuid": "4695cadc-ed35-4c85-9e09-e785024d38d0",
```

Comment:
Ensure form uuid matches that of the old HTML form "0248cd00-bc53-11ea-b3de-0242ac130004"